ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -49,6 +48,16 @@
 		3AE408501E1E280800F0FD83 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 3AE4084F1E1E280800F0FD83 /* LaunchScreen.storyboard */; };
 		832341BD1AAA6AB300B99B32 /* libRCTText.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 832341B51AAA6A8300B99B32 /* libRCTText.a */; };
 		BBDFD91C684046CB9FD0CD55 /* Entypo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 742B362B86E443C09A7F9FD9 /* Entypo.ttf */; };
+		A3EA8C86662742928E156E45 /* libTouchID.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 905F5155B95042B89DE3FADA /* libTouchID.a */; };
+		2FE5DD35D6C74CBDAA79E5E9 /* EvilIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = E7F4C82AC8DF4ADEBE2ED75C /* EvilIcons.ttf */; };
```

Comment:
We usually run `yarn react-native link $library`, where $library is (in this case) `react-native-touch-id`, so that we don't re-link all the stuff we've removed in the past.